.github/workflows/build-pr.yml

```diff
@@ -42,6 +42,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        java-version: [11, 17]
```

Comment:
Also need to update https://github.com/etcd-io/jetcd/blob/main/.github/workflows/build-main.yml

Reply:
I did think about this point.
Frankly, I was not sure what are the purposes of `build-main.yml` and `build-tag.yml`.
So, I focus on PR only. And that is why the title is `test with OpenJDK 17 on PR`.

Reply:
build tag was an attempt to automate the release process but it is not yet working